pkgs/development/tools/ocaml/js_of_ocaml/compiler-wasm.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  buildDunePackage,
+  binaryen,
+  cmdliner,
+  js_of_ocaml,
```

```suggestion
  js_of_ocaml-compiler,
```

and update the references below